spec/features/idv/previous_address_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature 'IdV with previous address filled in' do
+  include IdvHelper
+
+  let(:bad_zipcode) { '00000' }
+
+  it 'fails when either address has bad value, prefers current address in profile' do
+    user = sign_in_and_2fa_user
+    visit verify_session_path
+    fill_out_idv_form_ok
+    fill_out_idv_previous_address_fail
+    click_idv_continue
+
+    expect(current_path).to eq verify_session_path
+    expect(page).to have_selector("input[value='#{bad_zipcode}']")
+
+    fill_out_idv_previous_address_ok
+    fill_out_idv_form_fail
+    click_idv_continue
+
+    expect(current_path).to eq verify_session_path
+    expect(page).to have_selector("input[value='#{bad_zipcode}']")
```

Comment:
hmmm ok, maybe there is a way to group some of these steps into methods so that what is happening is more clear? I think these steps are really well-named as-is, but there are so many of them that it's kind of hard to read this spec.